epsagon/trace.py

```diff
@@ -476,6 +483,14 @@ def __init__(
                 )
         else:
             self.keys_to_ignore = []
+        if keys_to_allow:
+            self.keys_to_allow = [self._strip_key(x) for x in keys_to_allow]
+            if self.debug:
+                print(
```

Comment:
don't we have debug_log function here?
I think we added one here recently

Reply:
No